tools/sandbox/main.c

```diff
@@ -16,5 +18,14 @@ int main(int argc, char* argv[]) {
         fputs("Usage: please_sandbox command args...\n", stderr);
         return 1;
     }
-    return contain(&argv[1], true, true);
+
+    // Network namespace is sandboxed by default unless the `SANDBOX_NETWORK=0` env exists 
```

Comment:
I wonder if this would be clearer as `NOSANDBOX_NETWORK`? Setting it to 0 and having that be different from unset I think is a little unintuitive.

Reply:
We could have `SHARE_NETWORK` and default to unsharing it if it's unset

Reply:
`SHARE_NETWORK` is what I'm looking for! This was one of the moments where naming things was hard...